keyboards/idb/idb_60/idb_60.c

```diff
@@ -5,15 +5,26 @@ void keyboard_pre_init_kb(void) {
     setPinOutput(C5);
 }
 
-void led_set_kb(uint8_t usb_led) {
+// Capslock LED indicator
 
+void led_set_kb(uint8_t usb_led) {
     if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
-		_idb_60_caps_led_on();
-		} else {
-		_idb_60_caps_led_off();
+        _idb_60_caps_led_on();
+    } else {
+        _idb_60_caps_led_off();
     }
+    led_set_user(usb_led);
+}
```

Comment:
Since more things are changing here... 
```suggestion
bool led_update_kb(led_t led_state) {
    bool res = led_update_user(led_state);
    if (res) {
        if (led_state.caps_lock) {
            _idb_60_caps_led_on();
        } else {
            _idb_60_caps_led_off();
        }
    }
    return res;
}
```

Reply:
I'll have to test if that actually works this afternoon. Thanks

Reply:
Alright, that works. Sorry for the delay. Wanted to make sure there's no mixups sneaking in again on my side. Looking good otherwise?